e2e_tests/docker-compose.yml

```diff
@@ -80,7 +80,7 @@ networks:
     driver: bridge
     ipam:
       config:
-        - subnet: 10.10.0.0/24
+        - subnet: 11.10.0.0/24
 
 volumes:
-  liquiddata:
+  blockchain_volume:
```

Comment:
The rename isn't important. 
I did a quick check and couldn't find a way how to automatically remove the volumes and it looks like it's not possible because we use named volumes and not anonymous volumes.

Reply:
Thanks for checking. We can live with named volumes, but don't forget to delete them if you keep running the UI tests locally!